docs/crio.conf.5.md

```diff
@@ -25,143 +23,188 @@ No bare options are used. The format of TOML can be simplified to:
+  List to pass options to the storage driver. Please refer to containers-storage.conf(5) to see all available storage options.
 
-**storage_driver**=""
-  CRIO storage driver (default is "overlay")
+**file_locking**=true
+  If set to false, in-memory locking will be used instead of file-based locking.
 
-Note:
-  **overlay** and **overlay2** are the same driver
+**file_locking_path**="/runc/crio.lock"
+  Path to the lock file.
 
 
-**storage_option**=[]
-  CRIO storage driver option list (no default)
+## CRIO.API TABLE
+The `crio.api` table contains settings for the kubelet/gRPC interface.
 
-  Values:
+**listen**="/var/run/crio/crio.sock"
+  Path to AF_LOCAL socket on which CRI-O will listen.
 
-	"STORAGE_DRIVER.imagestore=/PATH",
+**stream_address**="127.0.0.1"
+  IP address on which the stream server will listen.
 
-	Paths to additional container image stores. These are read/only and are usually stored on remote network shares, based on overlay storage format.
-	storage_option=[ "overlay.imagestore=/mnt/overlay", ]
+**stream_port**="0"
+  The port on which the stream server will listen.
 
-	"STORAGE_DRIVER.size=SIZE"
+**stream_enable_tls**=false
+  Enable encrypted tls transport of the stream server.
 
-	Maximum size of a container image.  Default is 10GB. The size flag sets quota on the size of container images.
-	storage_option=[ "overlay.size=1G", ]
+**stream_tls_cert**=""
+  Path to the x509 certificate file used to serve the encrypted stream. This file can change, and CRI-O will automatically pick up the changes within 5 minutes.
 
-Note: Not all drivers support all options.
+**stream_tls_key**=""
+  Path to the key file used to serve the encrypted stream. This file can change, and CRI-O will automatically pick up the changes within 5 minutes.
 
-Note:  In order to use the **size** option for quota on *overlay* storage you must use the *xfs* file system.  The mount point that the *overlay* file system must be setup with the *pquota* flag at mount time. If you are setting up / to be used with quota, you have to modify the linux boot line in /etc/grubq2.conf and add the rootflags=pquota flag.
+**stream_tls_ca**=""
+  Path to the x509 CA(s) file used to verify and authenticate client communication with the encrypted stream. This file can change, and CRI-O will automatically pick up the changes within 5 minutes.
 
-Example:
-	linux16 /vmlinuz-4.12.13-300.fc26.x86_64 root=/dev/mapper/fedora-root ro rd.lvm.lv=fedora/root rd.lvm.lv=fedora/swap rhgb quiet LANG=en_US.UTF-8 rootflags=pquota
 
+## CRIO.RUNTIME TABLE
+The `crio.runtime` table contains settings pertaining to the OCI runtime used and options for how to set up and manage the OCI runtime.
 
-## CRIO.API TABLE
+**runtime**="/usr/bin/runc"
+  Path to the OCI compatible runtime used for trusted container workloads. This is a mandatory setting as this runtime will be the default one and will also be used for untrusted container workloads if `runtime_untrusted_workload` is not set.
```

Comment:
suggest "default one and" to "default and"

Reply:
:+1: